nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -13,7 +13,6 @@ define([
     'kb_service/client/catalog',
     'kb_service/client/narrativeMethodStore',
     'common/pythonInterop',
-    'common/utils',
```

Comment:
This one's slightly more complex as it imports both `common/utils` and `common/cellUtils` and uses their various non-overlapping functions separately. I kept the `cellUtils` part and adjusted the usage throughout the cell.